.changelog/unreleased/bug-fixes/1754-no-oversized-blocks.md

```diff
@@ -0,0 +1,2 @@
+- `[test]` Test app does not create oversized transaction set.
```

Comment:
Do we know of any external users/developers who make use of the test app? If not, there's no need for a changelog entry.